doc/usage/05_search/index.md

```diff
@@ -0,0 +1,97 @@
+# raptor search {#usage_search}
+
+[TOC]
+
+## Main Parameters
+
+### -​-index
+The path to the index. For partitioned indices, the suffix `_x`, where `x` is a number, must be omitted.
+
+### -​-query
+File containing queries.
+
+Supported file extensions are (possibly followed by bz2, gz, or bgzf):
+  * embl
+  * fasta
+  * fa
+  * fna
+  * ffn
+  * faa
+  * frn
+  * fas
+  * fastq
+  * fq
+  * genbank
+  * gb
+  * gbk
+  * sam
+
+### -​-output
+The output file name.
+
+<div class="tabbed">
+
+- <b class="tab-title">Format</b>
+  ```
+  ###<text>                     | Meta-information
+  ##<text>                      | Meta-information
+  #<number><tab><filepaths>     | Assigns each input file a number. Multiple filepaths are separated by a whitespace
+  #QUERY_NAME<tab>USER_BINS     | Header for the results
```

Comment:
Do you explain the term "user bins" somewhere?

Reply:
Not so important here, since it is just the header, and the output is (hopefully) well explained.

But it should go into the Quickstart